eng/Versions.props

```diff
@@ -25,8 +25,8 @@
     <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">false</StabilizePackageVersion>
     <DotNetFinalVersionKind Condition="'$(StabilizePackageVersion)' == 'true'">release</DotNetFinalVersionKind>
     <DefaultNetFxTargetFramework>net472</DefaultNetFxTargetFramework>
```

Comment:
Consider having a similar property for the default .NET core version. Say 

```xml
<DefaultNetCoreTargetFramework>net8.0</DefaultNetCoreTargetFramework>
```

This could let us avoid a few places where we're hard coding `net7.0 / net8.0`